CRM/Admin/Form/Setting/UF.php

```diff
@@ -84,6 +84,9 @@ public function buildQuickForm() {
       $tablePrefixes .= "\n];";
       $this->assign('tablePrefixes', $tablePrefixes);
     }
+    else {
+      $this->assign('tablePrefixes', FALSE);
```

Comment:
It might be clearer to place line 85 outside the loop so it is unconditionally assigning "tablePrefixes" to the template, and just ensure that variable is defined.